From 70e6cec07d86332f1aaf7a69bec75c7138306f6a Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Thu, 29 Jul 2021 14:20:14 +0900
Subject: [PATCH] hwfeatures: Enable hardware support also in FIPS mode.

* src/hwfeatures.c (_gcry_detect_hw_features): Remove skipping in FIPS
mode.

--

Reported-by: Jakub Jelen <jjelen@redhat.com>
GnuPG-bug-id: 5508
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
---
 src/hwfeatures.c | 3 ---
 1 file changed, 3 deletions(-)

Index: libgcrypt-1.9.4/src/hwfeatures.c
===================================================================
--- libgcrypt-1.9.4.orig/src/hwfeatures.c
+++ libgcrypt-1.9.4/src/hwfeatures.c
@@ -213,9 +213,6 @@ _gcry_detect_hw_features (void)
 {
   hw_features = 0;
 
-  if (fips_mode ())
-    return; /* Hardware support is not to be evaluated.  */
-
   parse_hwf_deny_file ();
 
 #if defined (HAVE_CPU_ARCH_X86)
